torchgeo/datasets/geo.py

```diff
@@ -578,6 +579,93 @@ def __str__(self) -> str:
     size: {len(self)}"""
 
 
+class VisionClassificationDataset(VisionDataset, ImageFolder):  # type: ignore[misc]
+    """Abstract base class for classification datasets lacking geospatial information.
+
+    This base class is designed for datasets with pre-defined image chips which
+    are separated into separate folders per class.
+    """
+
+    def __init__(
+        self,
+        root: str,
+        transforms: Optional[Callable[[Dict[str, Tensor]], Dict[str, Tensor]]] = None,
+        loader: Optional[Callable[[str], Any]] = None,
```

Comment:
Instead of making this optional why not set the default to `default_loader`?

Reply:
Done